app/models/miq_report/formatting.rb

```diff
@@ -191,6 +191,13 @@ def format_datetime(val, options)
     val.strftime(options[:format])
   end
 
+  def format_relative_date(val, options)
+    return val unless val.kind_of?(Time) || val.kind_of?(Date) || val.kind_of?(DateTime)
+    
+    helper = Module.new { |m| m.extend(ActionView::Helpers::DateHelper) }
+    "#{helper.time_ago_in_words(val)} ago"
```

Comment:
I would prefer we use the current mechanism. Will this work?

```suggestion
    "#{ApplicationController.helper.time_ago_in_words(val)} ago"
```

Reply:
So sorry, it is `ApplicationController.helpers` (I corrected my suggestion above)

Reply:
yes @kbrock it worked . pushed the change